includes/class-wc-taxjar-install.php

```diff
@@ -133,6 +141,7 @@ private static function get_schema() {
   created_datetime datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
   processed_datetime datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
   retry_count smallint(4) NOT NULL DEFAULT 0,
+  last_error longtext NOT NULL DEFAULT '',
```

Comment:
Do we anticipate the need for `longtext` here or would `text` be sufficient?

Reply:
You are right, text will be plenty sufficient for this column.